src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/CaseNotesApiService.kt

```diff
@@ -218,4 +244,39 @@ class CaseNotesApiService(
       )
     }
   }
+  suspend fun postCaseNote(prisonerId: String, casenotes: CaseNotesRequest): CaseNote {
+    val type = PATHWAY_PARENT_TYPE
+    val pathwayValues = PathwayMap.values()
+    val pathwayVal = pathwayValues.find { it.id == casenotes.pathway }
+    val subType = pathwayVal?.name.toString()
+    val prisonCode = findPrisonerPersonalDetails(prisonerId).prisonId
+    return offenderCaseNotesWebClientUserCredentials.post()
+      .uri("/case-notes/{offenderNo}", prisonerId)
+      .bodyValue(
+        mapOf(
+          "locationId" to prisonCode,
+          "type" to type,
+          "subType" to subType,
+          // "occurrenceDateTime" to LocalDate.now(),
```

Comment:
Should this be commented out?

Reply:
removed it. casenotes api  takes the current datetime